### DISCUSSION TITLE
Widget / Editor

### DISCUSSION THREAD

**Discussion by Banuk1616** [templaza/astroid-framework#544] [Ideas]
I use the JCE editor. For the workflow, it would be nice if Astroid would use the editor in the widgets that was primarily set up in Joomla.

**Comment by Chacapamac** [comment]
Yep, I second, **JCE USER FOREVER!**

**Comment by ghost** [comment]
Good idea. In the text widget it would be good to be able to use the same editor as in the Joomla backend.

**Comment by Chacapamac** [comment]
Well in fact ANY GOOD editor or component/Template/CMS should be able to use the same editor everywhere because it is easier for the people doing. We are talking about Content Management System after all.

**Comment by Brandocracy** [comment]
Totally agree! Can anyone tell me why the default Astroid template links are always _blank... they always open a new window?!? 
That seems a pretty standard feature to have as optional.

**Reply by sonvnn** [reply]
Can you detail the page or widget you get this issue? 

Thanks,

**Reply by Brandocracy** [reply]
Yes. So if I use an Accordion widget for example (or any widget with a text box), and I use the link tool in the editor it makes a link that opens a new page _blank... see screen shot.
![Screenshot 2024-08-20 at 5 51 17 PM](https://github.com/user-attachments/assets/2bee8dda-50d0-40e3-8567-ea22d19675d9)
